packages/global-styles/src/constants.ts

```diff
@@ -2,5 +2,6 @@ export const STYLE_PREVIEW_WIDTH = 248;
 export const STYLE_PREVIEW_HEIGHT = 152;
 export const STYLE_PREVIEW_COLOR_SWATCH_SIZE = 32;
 
+export const DEFAULT_VARIATION_TITLE = 'default';
```

Comment:
[nit] Could we make `DEFAULT_VARIATION_TITLE` and `DEFAULT_GLOBAL_STYLES_VARIATION_SLUG` naming consistent? Either one is fine with me!

Reply:
Sure!

Reply:
Updated! I renamed it to `DEFAULT_GLOBAL_STYLES_VARIATION_TITLE`